test/test_mock.py

```diff
@@ -0,0 +1,71 @@
+import yaml
+import github
+import gitlab
+from github.GithubException import UnknownObjectException
+from gitlab.exceptions import GitlabGetError
+
+
+with open('test_githubtest.yaml', 'r') as f:
+    config = yaml.load(f)
+
+
+# github
+
+def mock_get_user(user_name):
+    raise UnknownObjectException('args', 'kwargs')
+
+
+def mock_get_user_(user_name):
```

Comment:
One of them (without underscore) is to test the error condition and hence returns generated exception, whereas the one with underscore returns the mocked object (some value to continue execution of called method). Do you want me to rename them to make code more obvious?